implementations/micrometer-registry-otlp/src/test/java/io/micrometer/registry/otlp/OtlpMeterRegistryTest.java

```diff
@@ -52,6 +52,11 @@ abstract class OtlpMeterRegistryTest {
 
     abstract OtlpConfig otlpConfig();
 
+    @AfterEach
+    void clearRegistry() {
+        registry.clear();
```

Comment:
This shouldn't be necessary. Each test will run with a new instance of `OtlpMeterRegistry`

Reply:
removed it.